From df5af93de165435acfe1b91d49a824be68e8e27e Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:37:39 -0400
Subject: [ethernet] i40e: Update function formal parameters

Message-id: <1409658042-22689-99-git-send-email-sassmann@redhat.com>
Patchwork-id: 90292
O-Subject: [RHEL7.1 PATCH 098/281] i40e: Update function formal parameters
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Kevin Scott <kevin.c.scott@intel.com>

Change the formal parameters to remove argument which is no longer used.

Change-ID: I493b56eba74a1f396fa1b7d24c0a4acbe536b5bf
Signed-off-by: Kevin Scott <kevin.c.scott@intel.com>
Acked-by: Shannon Nelson <shannon.nelson@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit f4f94b94d6d6923ba7abfd1eedf931dcc1ccbbd5)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_common.c b/drivers/net/ethernet/intel/i40e/i40e_common.c
index e19df8f..55d029f 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_common.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_common.c
@@ -2102,8 +2102,8 @@ i40e_status i40e_aq_start_lldp(struct i40e_hw *hw,
  * @cmd_details: pointer to command details structure or NULL
  **/
 i40e_status i40e_aq_add_udp_tunnel(struct i40e_hw *hw,
-    u16 udp_port, u8 header_len,
-    u8 protocol_index, u8 *filter_index,
+    u16 udp_port, u8 protocol_index,
+    u8 *filter_index,
     struct i40e_asq_cmd_details *cmd_details)
 {
  struct i40e_aq_desc desc;
diff --git a/drivers/net/ethernet/intel/i40e/i40e_main.c b/drivers/net/ethernet/intel/i40e/i40e_main.c
index 153d7e5..acdcc25 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_main.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_main.c
@@ -5605,7 +5605,6 @@ static void i40e_handle_mdd_event(struct i40e_pf *pf)
  **/
 static void i40e_sync_vxlan_filters_subtask(struct i40e_pf *pf)
 {
- const int vxlan_hdr_qwords = 4;
  struct i40e_hw *hw = &pf->hw;
  i40e_status ret;
  u8 filter_index;
@@ -5623,7 +5622,6 @@ static void i40e_sync_vxlan_filters_subtask(struct i40e_pf *pf)
    port = pf->vxlan_ports[i];
    ret = port ?
          i40e_aq_add_udp_tunnel(hw, ntohs(port),
-           vxlan_hdr_qwords,
            I40E_AQC_TUNNEL_TYPE_VXLAN,
            &filter_index, NULL)
          : i40e_aq_del_udp_tunnel(hw, i, NULL);
diff --git a/drivers/net/ethernet/intel/i40e/i40e_prototype.h b/drivers/net/ethernet/intel/i40e/i40e_prototype.h
index 10652f6..d351832 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_prototype.h
+++ b/drivers/net/ethernet/intel/i40e/i40e_prototype.h
@@ -157,8 +157,8 @@ i40e_status i40e_aq_stop_lldp(struct i40e_hw *hw, bool shutdown_agent,
 i40e_status i40e_aq_start_lldp(struct i40e_hw *hw,
     struct i40e_asq_cmd_details *cmd_details);
 i40e_status i40e_aq_add_udp_tunnel(struct i40e_hw *hw,
-    u16 udp_port, u8 header_len,
-    u8 protocol_index, u8 *filter_index,
+    u16 udp_port, u8 protocol_index,
+    u8 *filter_index,
     struct i40e_asq_cmd_details *cmd_details);
 i40e_status i40e_aq_del_udp_tunnel(struct i40e_hw *hw, u8 index,
     struct i40e_asq_cmd_details *cmd_details);
-- 
1.7.1